shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitAnalysisResult.java

```diff
@@ -30,9 +30,18 @@
 
     private final List<LimitViolation> limitViolations = new ArrayList<>();
 
-    public ShortCircuitAnalysisResult(List<FaultResult> faultResults, List<LimitViolation> limitViolations) {
+    private List<FeederResult> feederResults;
+
+    public ShortCircuitAnalysisResult(List<FaultResult> faultResults, List<LimitViolation> limitViolations, List<FeederResult> feederResults) {
         this.faultResults.addAll(Objects.requireNonNull(faultResults));
         this.limitViolations.addAll(Objects.requireNonNull(limitViolations));
+        if (feederResults != null) {
+            this.feederResults = List.copyOf(feederResults);
+        }
```

Comment:
The constructor is called when `feederResulrts` is null, so this code is never used. What did you want to do?